test/ui-test/src/screens/StatusMainScreen.py

```diff
@@ -81,6 +83,20 @@ def open_wallet(self):
         click_obj_by_name(MainScreenComponents.WALLET_BUTTON.value)
     
     def user_is_online(self):
+        click_obj_by_name(MainScreenComponents.PROFILE_NAVBAR_BUTTON.value)
         profileButton = squish.waitForObject(getattr(names, MainScreenComponents.PROFILE_NAVBAR_BUTTON.value))
-        verify_equal(profileButton.badge.color.name, "#4ebc60", "The user is not online by default")
+        verify_equal(profileButton.badge.color.name, "#4ebc60", "The user is not online")
         
+    def user_is_offline(self):
+        profileButton = squish.waitForObject(getattr(names, MainScreenComponents.PROFILE_NAVBAR_BUTTON.value))
+        verify_equal(profileButton.badge.color.name, "#7f8990", "The user is not offline")
+        
+    def set_user_state_offline(self):
+        click_obj_by_name(MainScreenComponents.PROFILE_NAVBAR_BUTTON.value)
+        click_obj_by_name(MainScreenComponents.USERSTATUSMENU_INACTIVE_ACTION.value)
+        
+    def set_user_state_online(self):
+        click_obj_by_name(MainScreenComponents.PROFILE_NAVBAR_BUTTON.value)
+        click_obj_by_name(MainScreenComponents.USERSTATUSMENU_INACTIVE_ACTION.value)
```

Comment:
Is it here any reason to first click on inactive and then to active? Could it be directly a click on active?

Reply:
hey so the thing is that by default the user is set to online after signing up, thats why I set it to in offline first, else there is no effect?

Reply:
> hey so the thing is that by default the user is set to online after signing up, thats why I set it to in offline first, else there is no effect?

So, if I configure to be offline, after a restart I'll be online again?

Reply:
no no, the first time a user signs up he will be set to automaatic by default which he will be online.

Reply:
I changed it up a bit for it to make more sense. Please do check and approve when you can :)